com.unity.netcode.gameobjects/Runtime/Metrics/ClientIdProvider.cs

```diff
@@ -0,0 +1,21 @@
+﻿#if MULTIPLAYER_TOOLS
+using Unity.Multiplayer.Tools;
+
+namespace Unity.Netcode
+{
+    class ClientIdProvider : IClientIdProvider
```

Comment:
Mentioned this in the other PR but I think the direct line `SetConnectionId` is better, should this be removed?